internal/os/disk/api.go

```diff
@@ -194,52 +194,6 @@ func (DiskAPI) GetDiskNumber(disk syscall.Handle) (uint32, error) {
 	return devNum.DeviceNumber, err
 }
 
-func (DiskAPI) DiskHasPage83ID(disk syscall.Handle, matchID string) (bool, error) {
-	query := StoragePropertyQuery{}
-
-	bufferSize := uint32(4 * 1024)
-	buffer := make([]byte, 4*1024)
-	var size uint32
-	var n uint32
-	var m uint16
-
-	query.QueryType = PropertyStandardQuery
-	query.PropertyID = StorageDeviceIDProperty
-
-	querySize := uint32(unsafe.Sizeof(query.PropertyID)) + uint32(unsafe.Sizeof(query.QueryType)) + uint32(unsafe.Sizeof(query.Byte))
-	querySize = uint32(unsafe.Sizeof(query))
-	err := syscall.DeviceIoControl(disk, IOCTL_STORAGE_QUERY_PROPERTY, (*byte)(unsafe.Pointer(&query)), querySize, (*byte)(unsafe.Pointer(&buffer[0])), bufferSize, &size, nil)
-	if err != nil {
-		return false, fmt.Errorf("IOCTL_STORAGE_QUERY_PROPERTY failed: %v", err)
-	}
-
-	devIDDesc := (*StorageDeviceIDDescriptor)(unsafe.Pointer(&buffer[0]))
-
-	pID := (*StorageIdentifier)(unsafe.Pointer(&devIDDesc.Identifiers[0]))
-
-	page83ID := []byte{}
-	byteSize := unsafe.Sizeof(byte(0))
-	for n = 0; n < devIDDesc.NumberOfIdentifiers; n++ {
-		if pID.Association == StorageIDAssocDevice && (pID.CodeSet == StorageIDCodeSetBinary || pID.CodeSet == StorageIDCodeSetASCII) {
```

Comment:
I saw that this line is different in `GetDiskPage83ID`, @ddebroy might help us here